src/foam/u2/borders/CollapseBorder.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'CollapseBorder',
```

Comment:
How is this different than a FoldingSection?

Reply:
Is there a FoldingSection? I can't find it

Reply:
See foam/demos/u2/Border.js.